### PR TITLE
Feature/59 - Added firebase configuration 

### PR DESCRIPTION
**Pull request by saranshbhardwaj**

### CODE REVIEW

**Comment by ppator1** [comment]
Approved

File: BridgeCareApp/VuejsApp/src/components/DetailedReport.vue [hunk 1]
```diff
@@ -174,9 +191,9 @@
          */
         onRunSimulation() {
```
Comment (ARA-Trans) [inline BridgeCareApp/VuejsApp/src/components/DetailedReport.vue]: Get rid of this message.

Comment (saranshbhardwaj) [reply BridgeCareApp/VuejsApp/src/components/DetailedReport.vue]: Do we not want any modal popup before running the simulation?

Comment (ARA-Trans) [reply BridgeCareApp/VuejsApp/src/components/DetailedReport.vue]: No

Comment (saranshbhardwaj) [reply BridgeCareApp/VuejsApp/src/components/DetailedReport.vue]: As discussed, we are keeping the popup for now

File: BridgeCareApp/BridgeCare/DataAccessLayer/RunSimulation.cs [hunk 2]
```diff
@@ -19,13 +19,13 @@ public Task<string> Start(SimulationModel data)
...
-                Thread simulationThread = new Thread(new ThreadStart(start.CompileSimulation));
-                simulationThread.Start();
+                Thread simulationThread = new Thread(new ParameterizedThreadStart(start.CompileSimulation));
+                simulationThread.Start(true);
                 if (!simulationThread.IsAlive)
                 {
                     DBMgr.CloseConnection();
                 }
-                return Task.FromResult("Simulation completed successfully");
+                return Task.FromResult("Simulation is running in the background");
```
[the remark this answers is not in the document]
Comment (saranshbhardwaj) [reply BridgeCareApp/BridgeCare/DataAccessLayer/RunSimulation.cs]: changed

**Pull request merged**